app/controllers/groups/create.js

```diff
@@ -0,0 +1,38 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class extends Controller {
+
+  @service errorHandler;
+
+  GroupName ='';
```

Comment:
Sundaram, have you ever seen a JS variable, at least in this project starting with a capital letter?

Reply:
never 🤦🏻‍♂️